src/components/scenes/RequestScene.js

```diff
@@ -250,54 +250,59 @@ export class Request extends React.Component<Props, State> {
-            <InputAccessoryView backgroundColor={THEME.COLORS.OPAQUE_WHITE} nativeID={inputAccessoryViewID}>
-              <View style={styles.accessoryView}>
-                <TouchableOpacity style={styles.accessoryBtn} onPress={this.cancelFioMode}>
-                  <Text style={styles.accessoryText}>{this.state.isFioMode ? s.strings.string_cancel_cap : ''}</Text>
-                </TouchableOpacity>
-                <TouchableOpacity style={styles.accessoryBtn} onPress={this.nextFioMode}>
-                  <Text style={styles.accessoryText}>{this.state.isFioMode ? s.strings.string_next_capitalized : 'Done'}</Text>
-                </TouchableOpacity>
-              </View>
-            </InputAccessoryView>
-          ) : null}
-
-          <View style={styles.qrContainer}>
-            <QrCode data={this.state.encodedURI} size={qrSize} />
+        {keysOnlyMode !== true && (
+          <View style={styles.main}>
+            <ExchangedFlipInput
+              ref={this.flipInputRef}
+              headerText={flipInputHeaderText}
+              headerLogo={flipInputHeaderLogo}
+              primaryCurrencyInfo={primaryCurrencyInfo}
+              secondaryCurrencyInfo={secondaryCurrencyInfo}
+              exchangeSecondaryToPrimaryRatio={exchangeSecondaryToPrimaryRatio}
+              overridePrimaryExchangeAmount=""
+              forceUpdateGuiCounter={0}
+              onExchangeAmountChanged={this.onExchangeAmountChanged}
+              keyboardVisible={false}
+              color={THEME.COLORS.WHITE}
+              isFiatOnTop
+              isFocus={false}
+              onNext={this.onNext}
+              topReturnKeyType={this.state.isFioMode ? 'next' : 'done'}
+              inputAccessoryViewID={this.state.isFioMode ? inputAccessoryViewID : ''}
+            />
+
+            {Platform.OS === 'ios' ? (
+              <InputAccessoryView backgroundColor={THEME.COLORS.OPAQUE_WHITE} nativeID={inputAccessoryViewID}>
+                <View style={styles.accessoryView}>
+                  <TouchableOpacity style={styles.accessoryBtn} onPress={this.cancelFioMode}>
+                    <Text style={styles.accessoryText}>{this.state.isFioMode ? s.strings.string_cancel_cap : ''}</Text>
+                  </TouchableOpacity>
+                  <TouchableOpacity style={styles.accessoryBtn} onPress={this.nextFioMode}>
+                    <Text style={styles.accessoryText}>{this.state.isFioMode ? s.strings.string_next_capitalized : 'Done'}</Text>
+                  </TouchableOpacity>
+                </View>
+              </InputAccessoryView>
+            ) : null}
+
+            <View style={styles.qrContainer}>
+              <QrCode data={this.state.encodedURI} size={qrSize} />
+            </View>
+            <RequestStatus requestAddress={requestAddress} addressExplorer={addressExplorer} />
           </View>
-          <RequestStatus requestAddress={requestAddress} addressExplorer={addressExplorer} />
-        </View>
+        )}
```

Comment:
This should have an `else` case that tells the user the asset has been deprecated. ie

"Support for WAX has been deprecated. To continue using this wallet, extract the wallet private key and import it into a WAX supporting wallet."